data/schema/filters/abilities.cfg

```diff
@@ -7,11 +7,11 @@
 	{SIMPLE_KEY overwrite_specials string_list}
 	{SIMPLE_KEY apply_to string_list}
 	{SIMPLE_KEY active_on string_list}
-	{SIMPLE_KEY value s_range_list}
-	{SIMPLE_KEY add s_range_list}
-	{SIMPLE_KEY sub s_range_list}
-	{SIMPLE_KEY multiply s_real_range_list}
-	{SIMPLE_KEY divide s_real_range_list}
+	{SIMPLE_KEY value s_range_spe_list}
```

Comment:
What is spe? It looks like a spelling mistake… or is it an acronym?

Reply:
spe for special because is a special case, it don't find best for now.

Reply:
If we're using `~` as the separator character, I suggest `s_range_tilde_list`. I'd still prefer `..` or even the minus sign (with logic to resolve ambiguity).

Reply:
For what it's worth, in the regex it would look like this:
```
        [element]
            value="-?\d+"
        [/element]
        [element]
            value="-?\d+--?\d+"
        [/element]
        [element]
            value="-?\d+-infinity"
        [/element]
        [element]
            value="-infinity--?\d+"
        [/element]
        [element]
            value="-infinity-infinity"
        [/element]
```

Actually implementing that in the code is separate issue; and even if we did this, it might be worth making it a separate type, as we might want some range lists to allow negative values and others not to.

Reply:
> If we're using `~` as the separator character, I suggest `s_range_tilde_list`. I'd still prefer `..` or even the minus sign (with logic to resolve ambiguity).

for `..`impossible because two or more character in separator will be reject in Macos. It's between `~` and `_` what the choice willl be done. personally i prefer `~` for esthetical reason and also arlredy used like separator in halo but is not decided yet.

Reply:
I had a look at supporting `..`, will push to #7829 when it finishes the local test.